src/test/ui/borrowck/borrowck-suggest-extraction-of-subcomputation.nll.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0502]: cannot borrow `buckets` as immutable because it is also borrowed as mutable
+  --> $DIR/borrowck-suggest-extraction-of-subcomputation.rs:25:70
+   |
+LL |     buckets.slice_mut()[(key as usize).wrapping_add(22).wrapping_rem(buckets.sweep())] = 22;
+   |     -----------------------------------------------------------------^^^^^^^----------
```

Comment:
I wonder if there's an issue filed for this diagnostic change? It looks to be a worse version of what we have in the AST borrowck.

Reply:
Are you referring to the "really big span"?

Reply:
I'm not sure if there is an issue but I agree it's a diagnostic regression